near/omni-bridge/src/lib.rs

```diff
@@ -434,7 +446,14 @@ impl Contract {
     ) {
         if let Ok(signature) = call_result {
             if fee.is_zero() {
-                self.remove_transfer_message(message_payload.transfer_id);
+                let message = self.remove_transfer_message(message_payload.transfer_id);
+                let normalization_dust = message.amount.0 - message_payload.amount.0;
```

Comment:
What does this value represent, and why do we need it? We're subtracting the normalized value from the non-normalized one...

Reply:
The dust, or the difference between normalized and non-normalized values. This dust will be added to the fee on during the claim fee, but if the fee is zero then the dust will be stuck in the bridge contract.
The idea of storing this value was to be able to compare the total supply on different chains, but the benefit of this is debatable because the total supply is already hard to track if the the relayer decided not to claim the fee, so new tokens will not be minted.
So we can remove this logic and delegate this to the indexer.